frontend/content/locale/en.json

```diff
@@ -1,4 +1,22 @@
 {
   "hello": "Hello, world!",
-  "welcomeMessage": "Welcome to your internationalised page!"
+  "welcomeMessage": "Welcome to your internationalised page!",
+  "hello": "Hello, world!",
+  "welcomeMessage": "Welcome to your internationalised page!",
+  "featuresTitle": "Engineered for student success",
+  "feature1": "Well-designed",
+  "feature1msg": "Designed to be visually appealing with usability in mind",
+  "feature2": "Moderation",
+  "feature2msg":
+    "Built with all the tools you need to ensure responsible usage",
+  "feature3": "UofT Support",
+  "feature3msg":
+    "Integrated with Quercus & Shibboleth for a seamless instructor experience",
+  "feature4": "Accessible",
+  "feature4msg":
+    "Follows WAI-ARIA principles, enables students with disabilties",
+  "feature5": "LaTeX/Markdown",
+  "feature5msg": "Communicating math concepts is hard without symbols!",
+  "feature6": "Open Source",
+  "feature6msg": "Completely transparent & open to contribution from all!"
 }
```

Comment:
Sweeeeeeeet

Reply:
Suggestion: Can you add these messages to the other languages as well? You don't need to translate them, a simple `"feature4": "FR-Accessible"` ,` "feature4": "ES-Accessible"` is sufficient! It's so the console doesn't bork at us when we test in different languages.